src/codeu/chat/server/LogQueue.java

```diff
@@ -0,0 +1,52 @@
+package codeu.chat.server;
+
+
+
+import com.sun.scenario.effect.impl.sw.sse.SSEBlend_SRC_OUTPeer;
+
+import java.io.*;
+import java.util.TimerTask;
+import java.util.ArrayDeque;
+import java.util.Timer;
+
+/**
+ * Created by Lysander on 6/23/17.
+ */
+public class LogQueue {
+    File transactionLog;
```

Comment:
Do you mean the `server` package or the `Server` class?

If it's the former, then you should change the declaration of the class to remove the `public` keyword. Even after that, though, I would still recommend making these private and giving them accessor methods.

If it's the latter, then you could make the class it a `private static` nested class, declared inside `Server`. At that point, then I think it would be OK to skip accessors and access fields directly.